### PR TITLE
PTX-1254 Upgrade Openshift while keeping the same version of the px

### PR DESCRIPTION
**Pull request by thiguetta**
This PR is required to test torpedo against scheduler upgrades, in this specific case upgrade between openshift versions. It requires that these parameters are provided:
```
--sched-upgrade-hops=<comma separated value of ocp versions>
--scheduler=openshift
```

i.e 
```
--sched-upgrade-hops=4.5.30,4.6.20,4.7.2
--scheduler=openshift
```

### CODE REVIEW

**Comment by thiguetta** [comment]
regular dev-test passed here https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/883/console

File: drivers/scheduler/openshift/openshift.go [hunk 1]
```diff
@@ -137,6 +148,146 @@ func (k *openshift) updateSecurityContextConstraints(namespace string) error {
...
+	if versionReg.MatchString(version) {
+		return version, nil
+	}
+
+	ver, err := semver.Make(version)
+	if err != nil {
+		return "", err
+	}
+	channel := fmt.Sprintf("candidate-%d.%d", ver.Major, ver.Minor)
+	url := fmt.Sprintf("curl -sL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/%s | grep %s", channel, version)
```
Comment (Rohit-PX) [inline drivers/scheduler/openshift/openshift.go:238]: We can define this as a constant:
```
openshiftmirror="https://mirror.openshift.com/pub/openshift-v4/clients/ocp"
```
Will make the code more readable.

Comment (thiguetta) [reply drivers/scheduler/openshift/openshift.go]: done

**Comment by thiguetta** [comment]
tested here https://jenkins.portworx.dev/job/Torpedo/job/tp-ocp-upgrade/29/console

**Comment by thiguetta** [comment]
also tested here https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/893/console

File: drivers/volume/portworx/portworx.go [hunk 2]
```diff
@@ -946,15 +946,6 @@ func (d *portworx) WaitDriverUpOnNode(n node.Node, timeout time.Duration) error
...
 		logrus.Debugf("Getting node info: %s", n.Name)
-		for _, addr := range n.Addresses {
-			err := d.testAndSetEndpointUsingNodeIP(addr)
-			if err != nil {
-				return "", true, &ErrFailedToWaitForPx{
-					Node:  n,
-					Cause: fmt.Sprintf("failed to get node info [%s]. Err: %v", n.Name, err),
-				}
-			}
-		}
```
Comment (mborodin-px) [inline drivers/volume/portworx/portworx.go:957]: why do we remove this code?

Comment (thiguetta) [reply drivers/volume/portworx/portworx.go:957]: That piece of code is not necessary and affects test when using service type as Load Balancer

**Review by mborodin-px — APPROVED** [review]
LGTM

**Comment by thiguetta** [comment]
tested succeded here https://jenkins.portworx.dev/job/Torpedo/view/Torpedo%20NextPX/job/tp-ocp-upgrade/59/console

**Pull request merged**